semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/bad1/error.txt

```diff
@@ -0,0 +1,26 @@
+Traceback (most recent call last):
```

Comment:
Note that my paths being snapshotted here are okay, as this is marked as an expected failure and this issue is linked: https://github.com/returntocorp/semgrep/issues/678